src/processing/pokemon/components/cinematicMoves.ts

```diff
@@ -9,12 +9,7 @@ import * as _ from 'lodash';
 })
 export class CinematicMoves implements IComponent {
   Process(pokemon: Pokemon, rawPokemon: ItemTemplate): Pokemon {
-    const cinematicMoves = _
```

Comment:
This code you've deleted, fixed issue #6. Could you check it satisfies the requirements of issue #6 (no duplicate moves with Mew)  and issue #28.

There is a test for the #6 requirements, so if you run `npm run test` it should check that (as long as the test is written correctly)

Reply:
`_.map(_.uniq(rawPokemon.pokemonSettings.cinematicMoves), Util.SnakeCase2Identifyable)` does the job, but test still fails because it's expecting (Unique)pokemon.cinematicMoves.length to be equal as (non unique [raw])pokemon.cinematicMoves.length